### PR TITLE
add tweet and FB prompts

### LINKED ISSUE #52: Create social links
https://github.com/USGS-VIZLAB/water-use-15/blob/master/viz.yaml#L85

### PR DESCRIPTION
**Pull request by lindsaycarr**
Fixes #52.

image

capture

^ links and image cards don't show up because the site is not active

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -98,8 +98,8 @@ parameter:
   - 
     id: social-links
     github: https://github.com/USGS-VIZLAB/water-use-15
-    twitter: https://twitter.com/share?url=https://owi.usgs.gov/vizlab/water-use-15/?utm_source=twitter_button_share
-    facebookLink: http://www.facebook.com/sharer.php?u=https://owi.usgs.gov/vizlab/water-use-15/?utm_source=fb_button_share
+    twitter: https://twitter.com/share?url=https://owi.usgs.gov/vizlab/water-use-15/?utm_source=twitter_button_share%2F&text=The%20USGS%20studies%20water%20use%20in%20every%20county%20for%20the%20U.S.%20to%20manage%20current%20resources%20and%20prepare%20for%20future%20needs.%20Explore%20your%20county%20here&hashtags=water,usgs,wateruse
```
Comment (wdwatkins) [inline viz.yaml]: `every county **in** the U.S.`?

Comment (lindsaycarr) [reply viz.yaml]: would it be `water use FOR every county IN the US`? in followed by in sounds weird

Comment (wdwatkins) [reply viz.yaml]: Yeah, `water use for every county OF the US` maybe?

Comment (wdwatkins) [reply viz.yaml]: I mean `water use in every county of the US`

**Comment by lindsaycarr** [comment]
Alright, these tweets were approved by Cheryl and Nancy (thanks for the grammar check David 👍) and I've tested to make sure they do in fact work locally. I'm going to merge.

**Pull request merged**